### ISSUE TITLE
Storybook crashes when React components are passed to addon knobs select.

### ISSUE THREAD

**Issue by sami616** [storybookjs/storybook#10422]
This seems to crash storybook when i select a value from the select input:

```typescript
import { select } from '@storybook/addon-knobs';
import { Icon } from './Icon'
// 👇🏼These icons are react components
import { IconOne, IconTwo } from './icons';  

<Icon icon={select('icon', { IconOne, IconTwo }, IconOne )} />
```

Is this possible with addon knobs?

**Comment by shilman** [comment]
Hi gang, We’ve just released addon-controls in 6.0-beta!

Controls are portable, auto-generated knobs that are intended to replace addon-knobs long term.

- [Controls README](https://github.com/storybookjs/storybook/tree/next/addons/controls)
- [Migration instructions](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)

Please upgrade and try them out today. Thanks for your help and support getting this stable for release!

**Comment by shilman** [comment]
For anybody who is interested in Controls but don't know where to start, I've created a quick & dirty step-by-step walkthrough to go from a fresh CRA project to a working demo. Check it out:

=> [Storybook Controls w/ CRA & TypeScript](https://gist.github.com/shilman/69c1dd41a466bae137cc88bd2c6ef487)

There are also some "knobs to controls" migration docs in the Controls README:

=> [How do I migrate from addon-knobs?](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)
